task/02-numbers-tasks.js

```diff
@@ -127,8 +130,8 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
-}
+    return value.toString()[ value.toString().length - 1 ];
+  }
 
```

Comment:
should return number, try to use the division remainder
https://www.w3schools.com/js/js_arithmetic.asp